pkg/aws/permissions.go

```diff
@@ -18,8 +19,10 @@ type SimulateParams struct {
 func (c *awsClient) ValidateSCP(target *string, policies map[string]*cmv1.AWSSTSPolicy) (bool, error) {
 	policyDetails := GetPolicyDetails(policies, "osd_scp_policy")
 
+	region := c.GetRegion()
```

Comment:
@den-rgb I don't think there's any need to assign the `region` variable. The `c.getRegion()` function call can be moved onto the definition of the `SimulateParams` struct e.g:

```
sParams := &SimulateParams{
		Region: c.GetRegion(),
	}
```